# Overview

Call

	python example.py

should result in two instances of logging to stderr. Only one occurs.
